desktop-environment.el

```diff
@@ -164,6 +164,23 @@ portion of the screen."
   :type 'string)
 
 
+
+;;; Customization - keyboard layout
+
+(defcustom desktop-environment-keyboard-layouts '("us" "us -variant dvorak" "cz")
+  "List of layouts to toggle between with.
+
+This is prepended by \"setxkbmap -layout\".
+The elements of this list must be valid xkb keymaps.
+List of all valid keymaps: \"man xkeyboard-config\""
```

Comment:
```
See (man \"xkeyboard-config\") for a list of all valid keymaps.
```

This makes it possible for the user to just press `C-x C-e` and get the man page :-).